deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -85,8 +85,8 @@ data:
         cache_size  "10000"
         cache_ttl  "3600"
         cache_refresh "1800"
-        core_api_versions [v1]
-        api_groups [apps/v1 extensions/v1beta1]
+        core_api_versions "[v1]"
+        api_groups "[apps/v1 extensions/v1beta1]"
```

Comment:
@samjsong do we need quotes? Also, inside the brackets, the two items are with no quotes and  no comma

Reply:
cc @perk-sumo

Reply:
that's interesting, can you try a non-helm install and see if this works?